### DISCUSSION TITLE
v0.2.3 Is Excellent

### DISCUSSION THREAD

**Discussion by gary-hammer** [bellrichm/weewx-jas#65] [General]
Thanks for all the work you have put in.
0.2.3 now has replaced what I was using on my tablet that is a 'console'.
My wife loves appTemp and with the addition of max_decimals the skin is perfect for our use.

**Comment by bellrichm** [comment]
Thanks for the kind words. Glad you are finding it useful. I wrote it because I couldn’t find any skin that looked decent on my tablet.
All feedback and ideas are welcome. Just no more defects :)
